examples/step-13/step-13.cc

```diff
@@ -1403,21 +1366,23 @@ namespace Step13
         // Now solve the problem on the present grid, and run the evaluators
         // on it. The long type name of iterators into the list is a little
         // annoying, but could be shortened by a typedef, if so desired.
-        solver.solve_problem ();
+        solver.solve_problem();
 
-        for (typename std::list<Evaluation::EvaluationBase<dim> *>::const_iterator
-             i = postprocessor_list.begin();
-             i != postprocessor_list.end(); ++i)
+        for (typename std::list<
+               Evaluation::EvaluationBase<dim> *>::const_iterator i =
+               postprocessor_list.begin();
```

Comment:
This is also likely going to go away with more horizontal space, but it's a bit difficult to discern here what is template argument, what is variable name, and what is initializer because the template argument and the initializer are indented the same. The original was clearly easier to read.

Reply:
What about `for (auto i : postprocessor_list)`? :tongue: